antenna/configlib.py

```diff
@@ -180,38 +193,49 @@ def _initialize_environments(self):
         # Add OS environment
         envs.append(ConfigOSEnv())
 
-        # Add .ini environment
-        fn = os.environ.get('ANTENNA_INI', 'antenna.ini')
-        if os.sep not in fn:
-            fn = os.path.join(os.getcwd(), fn)
+        # Add .ini environment if they specify ANTENNA_INI in the environment
+        fn = os.environ.get('ANTENNA_INI')
+        if fn:
+            if os.sep not in fn:
+                fn = os.path.join(os.getcwd(), fn)
```

Comment:
Not really a nit, but isn't that what `open()` does by default. If you type `open('file.ini')` it will implicitly convert that to `open(os.path.join(os.getcwd(), 'file.ini'))`. 
We could use this to our advantage and just try to read the file and if it doesn't exist, consider excepting an IOError. That could potentially kill two birds with one stone. I.e. you'd end up in the same IOError exception if the files doesn't exist OR can't be read.